test/fixtures/node.yml

```diff
@@ -236,3 +236,25 @@ method:
   type: "page"
   cached_likes: 0
   path: "/wiki/spectrometer"
+
+node1:
+  nid: 21
+  uid: 8
+  title: "Method page"
+  created: <%= 4.days.ago %>
+  changed: <%= 4.days.ago %>
+  status: 1
+  type: "page"
+  cached_likes: 0
+  path: "/wiki/spectrometer"
+
+node2:
+  nid: 22
+  uid: 8
+  title: "Method page"
+  created: <%= Time.now.to_i %>
+  changed: <%= Time.now.to_i %>
+  status: 1
+  type: "page"
+  cached_likes: 0
+  path: "/wiki/spectrometer"
```

Comment:
Looks like you have two nodes with the same path and title. Can you choose another example to work from, like `/wiki/kite-mapping`?